interface/modules/custom_modules/oe-module-faxsms/contact.php

```diff
@@ -0,0 +1,220 @@
+<?php
```

Comment:
Is there a mechanism on the directly runnable php scripts that check to ensure the module has been enabled? (ie. stop scripts very early on if the module has not been enabled)